superglue/lib/utils/request.ts

```diff
@@ -36,9 +36,20 @@ export function validateResponse(args) {
 export function handleServerErrors(args) {
   const { rsp } = args
   if (!rsp.ok) {
-    const error = new Error(rsp.statusText)
-    error.response = rsp
-    throw error
+    if (rsp.status === 406) {
+      console.warn(
+          "Superglue encountered a 406 Not Acceptable response. This can happen if you used respond_to and didn't specify format.json in the block. Try adding it to your respond_to. For example:\n\n" +
+          "respond_to do |format|\n" +
+          "  format.html\n" +
+          "  format.json\n" +
+          "  format.csv\n" +
+          "end"
+      )
+    } else {
+      const error = new Error(rsp.statusText)
+      error.response = rsp
+      throw error
+    }
```

Comment:
Thinking about this some more, I think we should console.error, but continue to throw so we don't swallow the error for the user. I'd like to make this small change:


```suggestion
    if (rsp.status === 406) {
      console.error(
          "Superglue encountered a 406 Not Acceptable response. This can happen if you used respond_to and didn't specify format.json in the block. Try adding it to your respond_to. For example:\n\n" +
          "respond_to do |format|\n" +
          "  format.html\n" +
          "  format.json\n" +
          "  format.csv\n" +
          "end"
      )
    } 
    const error = new Error(rsp.statusText)
    error.response = rsp
    throw error
```

Reply:
I agree, thanks for the suggestion!